setup/setup.iss

```diff
@@ -216,7 +220,10 @@ begin
     TablesVppSHA1 := GetSHA1OfFile(ExtractFileDir(GameExePath) + '\tables.vpp');
     case TablesVppSHA1 of
         'ded5e1b5932f47044ba760699d5931fda6bdc8ba':
-            // 1.20 NA - this is directly supported version
+            // 1.10/1.20 NA - this is directly supported version
+            begin end;
+        '2541f5565517ea38dfb9645d432eb3d11a15ac35':

```

Comment:
Perhaps instead of adding more hardcoded checksums here it would be better to keep it as simple as possible and  handle the "1.21 GR GOG" tables.vpp only and ignore all other versions? This way we wouldn't need to worry about German and French checksums for this file.
I think I wrote that code when patches were applied immediately instead of being queued and it made more sense back then. Now it makes no sense to do anything with tables.vpp before 1.20 patch is applied.

Reply:
That would really simplify things. The patch would fail to install with an error window in the case that `tables.vpp` or any other files to be patched were modified, so these checks are not really needed. Do you know if the French and German retail versions use the same `tables.vpp` as the English version?

Reply:
If you do have those copies in your possession, do you also mind updating them to 1.10 and providing the SHA1 hashes for the `RF.exe` files? German and French 1.10 are currently missing from the version check for the patching procedure and I’d be thrilled to include them.

Reply:
@Mystyled are you going to remove unnecessary verification for unknown tables.vpp hashes? I really believe we don't need it in the installer

Reply:
Yes, I have removed them from my WIP copy of the script. Only the German GOG table is checked for now